src/test/java/com/gooddata/AbstractGoodDataAT.java

```diff
@@ -72,4 +75,11 @@ public static void removeProjectAndLogout() {
         }
         gd.logout();
     }
+
+    @AfterSuite
+    public void removeAccount() {
```

Comment:
not sure whether this is a good idea. firstly - you removed a test. secondly this after method is executed always even when tests are not run by domain admin